tracing-attributes/tests/instrument.rs

```diff
@@ -96,6 +96,9 @@ fn skip() {
     #[instrument(target = "my_target", level = "debug", skip(_arg2, _arg3))]
     fn my_fn(arg1: usize, _arg2: UnDebug, _arg3: UnDebug) {}
 
+    #[instrument(target = "my_target", level = "debug", skip_all)]
+    fn my_fn2(_arg1: usize, _arg2: UnDebug, _arg3: UnDebug) {}
```

Comment:
it looks like the test never actually calls this function or makes assertions about the span it records, so we're just testing that this compiles. which is _probably_ enough, but it might be nice if we actually called it as well.

Reply:
Yeah I was wondering if there were some existing skip tests for that.

Reply:
> Yeah I was wondering if there were some existing skip tests for that.

This is where the test calls `my_fn` and asserts that a field is observed for the argument that is not skipped, but no fields are observed for the arguments that are not skipped: https://github.com/tokio-rs/tracing/blob/f28a62aa34d5e781c467c28f7ed9a94f33279f12/tracing-attributes/tests/instrument.rs#L107-L133

we would want to do the same thing, except without expecting the `arg1` field.